v4-client-js/src/clients/native.ts

```diff
@@ -1306,6 +1312,34 @@ export async function signCompliancePayload(payload: string): Promise<string> {
   }
 }
 
+export async function signPushNotificationTokenRegistrationPayload(payload: string): Promise<string> {
```

Comment:
should we just have a generic sign payload function? is there specific handling needed for either the push case or compliance case?